docs/TUTORIAL.md

```diff
@@ -1,3 +1,310 @@
 # Tutorial
 
-Coming soon!
+The **ksonnet** [readme][readme] 
+shows you how to create a default 
+`deployment.json` file that lets you deploy an nginx container 
+to an existing Kubernetes cluster. This tutorial shows you how to:
+
+* Modify the deployment using the built-in **ksonnet** mixins
+* Write a custom mixin library for Fluentd 
+* Add the custom Fluentd mixin to your application deployment
+
+For more information about Fluentd, see [the Fluentd website](http://www.fluentd.org/).
+
+## Prerequisites
+
+This tutorial assumes that you have performed the following 
+tasks. For details, see the [readme][readme].
+
+* Installed **Jsonnet**, version 0.9.4 or later
+* Cloned the **ksonnet** repository locally
+* Installed and configured the VisualStudio Code extension 
+(optional)
+* Created a test Kubernetes cluster
+
+NOTE: All import paths are relative to the root of the 
+**ksonnet** repository.
+
+## Add a service to the deployment configuration
```

Comment:
For new the big thing in this draft is: it sends like if we use normal json for the service and the deployment, then we can skip right to explaining how the missions work.